src/reanimated2/mutables.ts

```diff
@@ -103,9 +84,9 @@ export function makeMutable<Value>(
         })();
       }
     },
-    get value() {
-      if (syncDataHolder) {
-        return NativeReanimatedModule.getDataSynchronously(syncDataHolder);
+    get value(): Value {
+      if (!SHOULD_BE_USE_WEB) {
+        return uiValueGetter(mutable);
       }
       return value;
```

Comment:
Let's do the same check as in `set value(newValue)`
```suggestion
      if (SHOULD_BE_USE_WEB) {
        return value;
      }
      return uiValueGetter(mutable);
```